packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -45,8 +53,12 @@ export class DocumentStorageService implements IDocumentStorageService {
             requestVersion = versions[0];
         }
 
-        const tree = await this.manager.getTree(requestVersion.treeId);
-        return buildHierarchy(tree, this.blobsShaCache);
+        const rawTree = await this.manager.getTree(requestVersion.treeId);
+        const tree = buildHierarchy(rawTree, this.blobsShaCache);
+
+        const logTailHash = ".logTail" in tree.trees ? tree.trees[".logTail"].blobs.logTail : undefined;
+        this._logTail = logTailHash ? await readAndParse<ISequencedDocumentMessage[]>(this, logTailHash) : [];
```

Comment:
Can we defer this call until getDeltas() are called? We ideally don't want to wait for this before summaries are returned to client.